src/redisearch_rs/encode_decode/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "encode_decode"
+version.workspace = true
+edition.workspace = true
+license-file.workspace = true
+publish.workspace = true
+
+[lints]
+workspace = true
+
+[lib]
+crate-type = ["staticlib", "rlib"]
```

Comment:
Just confirming: why is this a static library on its own?

Reply:
How would dynamic linking work? The library will have to be installed in a canonical location for the dynamic linker to load it at runtime then. As a static library, it just becomes part of the Redisearch binary.

Reply:
I mean it does not go through the `redisearch_rs` like all the other exposed functions. I might be missing something here, but don't remember this being needed on the trie crate :thinking:

Reply:
Ah you're right. The thing is that this work started before we had a `redisearch_rs` and `trie` did indeed this this (and where I copied this from) before that (see commit a0fb945982559db5d6841f8172259f30bd30692e). I'll update..